bindings/python/test/pandas_types/test_binary.py

```diff
@@ -14,10 +14,14 @@
 import numpy as np
 import pytest
 from bson import Binary
-from pandas.tests.extension import base
 
 from pymongoarrow.pandas_types import PandasBinary, PandasBinaryArray
 
+try:
+    from pandas.tests.extension import base
+except ImportError:
+    pytest.skip("skipping pandas tests", allow_module_level=True)
```

Comment:
Could you add some thing like "To include, create environment like so: `uv sync --dev --extra test --extra test-polars --extra test-pandas`"